examples/lambda/function.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: lambda.aws.crossplane.io/v1alpha1
+kind: Function
+metadata:
+  name: test-function
+spec:
+  forProvider:
+    code:
+      s3Bucket: "my-s3-bucket"
+      s3Key: "prefix/path_to_key.zip"
+    environment:
+      variables:
+        envKey: envValue
+    functionName: "test-function"
+    handler: "index.handler"
+    role: arn:aws:iam::123456789012:role/test-function-role
```

```suggestion
    roleRef:
      name: somerole
```

We try to use references to other example YAMLs.